mods/d2k/rules/structures.yaml

```diff
@@ -16,8 +16,7 @@
 		RemoveInstead: true
 	Buildable:
 		Queue: Building
-		Prerequisites: construction_yard
-		BuildPaletteOrder: 10
+		BuildPaletteOrder: 10 
```

Comment:
@GraionDilach: I got some warning about CLRF being converted to LF, so it would "keep original line endings". I ignored it because im not sure what it means^^ Is there a problem?
I think i also did enter an " " to be able to save the file, because my git (+ext) at one time did not show my changes in the git (+ext) editor. If you mark the second line with your cursor, you see the additional " ". thats why.